src/rpg/Fighter.java

```diff
@@ -0,0 +1,69 @@
+package rpg;
+
+import engine.engineException.InvalidLocusException;
+import engine.model.Individual;
+import rpg.items.Equipment;
+import rpg.items.EquipmentStash;
+
+public class Fighter implements Individual {
+    private final static double maxHeight = 2.0;
+    private final static double minHeight = 1.3;
+
+    private double height;
+    private Equipment[] equipment;
+
+    public Fighter(double height, Equipment[] equipment) {
+        this.height = height;
+        this.equipment = equipment;
+    }
+
+    @Override
+    public Individual replicate() {
+        return new Fighter(this.height, this.equipment);
```

Comment:
Ojo que cuando replicás le estás dando el mismo arreglo, no una copia, por lo que cuando mutes un equipamento también va a mutar el del otro. Tenés que hacer una copia del arreglo (`this.equipment = equipment.clone()`).